### PR TITLE
Feature/sample list meta

### PR DESCRIPTION
**Pull request by wolfbiter**
add loadMeta to loadSampleList. display sample titles instead of ids. display sample meta in sample detail view

### CODE REVIEW

File: lib/number-utils.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+function clamp (min, n, max) {
+  if (n < min) {
+    return min
+  } else if (n > max) {
+    return max
+  } else {
+    return n
+  }
+}
```
Comment (wolfbiter) [inline lib/number-utils.js:21]: these are ported from old linx... if you have better suggestions open to them

File: mixes/containers/mix.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ class MixContainer extends React.Component {
     if (!mix) { return null }
 
     console.log('mix', mix)
-    const title = isLoading
+    const $title = isLoading
```
Comment (wolfbiter) [inline mixes/containers/mix.js]: how do you normally name react element variables as opposed to normal things like strings?

File: samples/getters.js [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
 const getSamplesError = (state) => state.samples.error
 
+const getSamples = Getter(
+  getSamplesRecords,
+  getMetas,
+  (samples, metas) => omitBy(mapValues(samples, (sample, sampleId) => ({
+    ...sample,
+    meta: metas[sampleId] || {}
+  })), isNil)
+)
```
Comment (wolfbiter) [inline samples/getters.js:19]: mimic naming and style of `getMixList` in `mixes/getters`

File: mixes/containers/mix.js (hunk 2 above)
Comment (ahdinosaur) [reply mixes/containers/mix.js]: idk, i don't name them any differently. isn't this just a boolean? if it's an element, i often name them like `loadingElement`.

**Review by ahdinosaur — APPROVED** [review]

File: dev.js [hunk 4]
```diff
@@ -3,12 +3,12 @@ const chokidar = require('chokidar')
...
+  .watch('main.js')
+  .on('change', () => electron.restart())
 
-chokidar.watch('**/*.(js|json|html)', {
-  ignored: '*/node_modules/*'
-})
-.on('change', () => electron.reload())
+chokidar
+  .watch('**/*.(js|json|html)', {
+    ignored: '**/data/**'
```
Comment (wolfbiter) [inline dev.js:12]: fix bug where saving meta was reloading the app when in dev mode. lol.

also it seems relevant to reload when node_modules changes. i also couldnt see a way to add multiple `ignores`. thoughts?

**Pull request merged**